### ISSUE TITLE
LogcatMonitor shouldn't use the default buffer size to write outputs

### ISSUE THREAD

**Issue by bashi** [catapult-project/catapult#1878]
It seems that writing a line doesm't always flush buffer. I believe this is the cause of http://crbug.com/575482

**Comment by jbudorick** [comment]
Beware that nothing has been switched over to the catapult version of LogcatMonitor yet, though I think that should be completed by the end of next week.

**Comment by bashi** [comment]
Yeah, actually I tried to create a CL against src/build/android/devil first, but `git cl upload` rejected it :)
Here is a fix attempt. https://codereview.chromium.org/1567303002/
